msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -49,46 +62,226 @@ See the following video to learn how to embed Share to Teams button:
+       await window.shareToMicrosoftTeams.renderButtons({elements: [shareButton]});
+   }
+   ```
+
+---
+
+### Share content in meetings
+
+Share to Teams from web apps now allows users to share content to a live or an upcoming meeting. By enabling content sharing to meetings, you can allow users to share information that creates a collaborative workspace within the meeting. For example, users can share workboards from a web app directly onto the meeting stage, facilitating a collective effort to gather and organize ideas.
+
+Here's how the shared web content appears in the meeting stage:
+
+:::image type="content" source="../../assets/images/share-to-teams/share-meeting-live.png" alt-text="Screenshot shows app shared to the teams meeting stage.":::
+
+Before you enable share content to meetings, you must ensure the following:
+
+1. A [Microsoft Teams app with a tab](../../apps-in-teams-meetings/build-tabs-for-meeting.md) that supports meeting side panel and Share to Stage APIs.
+
+2. To display the content on the meeting stage, the meeting app must support [Share to Stage APIs](../../apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md) with two primary requirements:
+
+    * The [app manifest](../../resources/schema/manifest-schema.md) that supports both `meetingStage` and `meetingSidePanel` configured as frame contexts. If not configured, meeting participants might not be able to view the content on stage.
+    * The app that supports `MeetingStage.Write.Chat` permissions for taking control of the stage. This is a read permission requirement.
+
+> [!NOTE]
+>
+> * If the app doesn't have meeting stage and side panel capabilities, the content aren't stored and the messages are just sent in the regular meeting chat.
+> * If app installation is blocked by the admin, then the app is added to the chat and not the meeting of the user.
+> Share to Teams in meeting is supported only for instant tab scenarios and specific to Teams.
+
+**Enable Share to Teams for meeting**
+
+1. Add the `launcher.js` script on your webpage.
+
+    ```html
+    <script async defer src="https://teams.microsoft.com/share/launcher.js"></script>
     ```
 
-1. If the shared link requires user authentication, and the URL preview from your link to be shared doesn't render well in Teams, then you can disable the URL preview by adding the `data-preview` attribute set to `false`.
+1. Add an HTML element on your webpage with the `teams-share-button` in the `class` attribute and the link to share in the `data-href` attribute. To enable your users to share content in meetings from Share to Teams, add `allow-share-in-meeting` attribute and set it to `true`.
 
     ```html
-    <div
-      class="teams-share-button"
-      data-href="https://<link-to-be-shared>"
-      data-preview="false">
-    </div>
+        <div
+        class="teams-share-button"
+        data-href="https://<link-to-be-shared>"
+        data-allow-share-in-meeting="<true>"
+        >
```

Comment:
They will also need to pass 
data-app-id="<app-id>"

Reply:
Updated the code.
       <div
        class="teams-share-button"
        data-href="https://<link-to-be-shared>"
        data-allow-share-in-meeting="true"
        data-app-id="<app-id>"
        >
        </div>